json-schema/json-schema/src/test/scala/endpoints/HashingTest.scala

```diff
@@ -0,0 +1,16 @@
+package endpoints
+
+import org.scalacheck.Properties
+import org.scalacheck.Prop.forAll
+
+object HashingTest extends Properties("Hashing") {
+
+  property("hash") = forAll { (x: Int, y: Int, s: String, t: String) =>
+    val oneValue =
+      (x == y) == (Hashing.hash(x) == Hashing.hash(y))
+    val twoValues =
+      ((x, s) == (y, t)) == (Hashing.hash(x, s) == Hashing.hash(y, t))
```

Comment:
It seems that this syntax doesn’t work with Scala 2.

```suggestion
      ((x, s) == ((y, t))) == (Hashing.hash(x, s) == Hashing.hash(y, t))
```